### ISSUE TITLE
If about:blank is navigated to as the first URL, updated CoreWebView2 settings are not applied

### ISSUE THREAD

**Issue by dqwork** [MicrosoftEdge/WebView2Feedback#1049]
**Description**
After the corewebview2 property has been initialised, if the first url that is navigated to is 'about:blank' any settings changes that were made in CoreWebView2InitializationCompleted are not applied.

The documentation does state that changes to to the corewebview's settings require a top level navigation, but that raises two issues

1. I navigated to about:blank since I don't have a sensible defaut.. why does that not update them.
2. In general how is one supposed to set up the CoreWebView how you desire if you can't access that Settings Prop until the core view is initiliased and at that point you have to navigate to a valid url for it to update

**Version**
SDK: 1.0.774.44
Runtime:  89.0.774.45 (fixed version) and Evergreen runtime (version string 89.0.774.48)
Framework: Winforms .NET Framework 4.6.2
OS: Windows 10

**Repro Steps**
In the event handler for CoreWebView2InitializationCompleted set the corewebviews settings to not show context menus'
webView.CoreWebView2.Settings.AreDefaultContextMenusEnabled = false;
Then navigate to "about:blank" either by setting the webView source or directly called the corewebview's Navigate method. This can be in the event handler, or invoked by the end user with a button click, it doesn't matter, if the first url visited is about:blank the settings are not applied.

You will see that the result webview if you right click you get a context menu

Same applies to AreDevToolsEnabled

**Code**

```
using Microsoft.Web.WebView2.Core;
using System;
using System.Diagnostics;
using System.IO;
using System.Reflection;
using System.Windows.Forms;

namespace WebView2_Testing
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            webView.CoreWebView2InitializationCompleted += WebView_CoreWebView2Ready;
            InitializeAsync();
        }

        private void WebView_CoreWebView2Ready(object sender, EventArgs e)
        {
            webView.CoreWebView2.Settings.AreDefaultContextMenusEnabled = false;
            webView.CoreWebView2.Settings.AreDevToolsEnabled = false;
            webView.CoreWebView2.Settings.IsZoomControlEnabled = false;
            webView.Source = new Uri("about:blank");
        }

        private async void InitializeAsync()
        {
            await webView.EnsureCoreWebView2Async(null);
        }

        private void refreshBtn_Click(object sender, EventArgs e)
        {
            webView.CoreWebView2.Navigate("about:blank");
            //webView.Reload();
        }
    }
}
```

[AB#32107126](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/32107126)

**Comment by champnic** [comment]
Thanks for the bug report @dqwork - I've opened an item on our backlog to take a look.

My first instinct is that perhaps navigating to "about:blank" if you are already on "about:blank" doesn't count as a navigation. Can you try navigating to somewhere else (like good ol' "https://bing.com"), set the settings, and then try navigating to "about:blank" to see if they get applied?

**Comment by dqwork** [comment]
'Revisiting' the same page probably doesn't count as navigation - that seems fair enough to me.

However it leaves one big question - How to create a control set its props and then let the user start at a blank page - that seems like a reasonable 'workflow' right? Currently to do this I'd have to navigate to a website and then back to about:blank to make sure my control was properly configured, since I can't configure the core webwiew until its created and it seems to be created already 'navigated' to about:blank.

**Comment by champnic** [comment]
Can you try reloading? I saw you had the code commented out to reload - I'm not sure if reload counts as a navigation for the purposes of applying settings. I think it probably should if it doesn't.

**Comment by clshan-13** [comment]
@champnic @dqwork Hi, i wonder if this problem has been solved？Thanks

**Comment by champnic** [comment]
I'm not sure if @dqwork was able to test 'Reload' or found some other workaround, but we haven't changed the behavior here. We will close out issues once we've shipped a fix.

**Comment by champnic** [comment]
This is fixed in runtimes 93.0.961.34+. Thanks!
